pkg/orchestration/wiring/internaldns/plugin.go

```diff
@@ -17,8 +18,6 @@ import (
 
 const (
 	ResourceType                   voyager.ResourceType = "InternalDNS"
```

Comment:
Remove from here as it is in the `api` subpackage now

Reply:
fixed